crate/pages/blog/Snowsprint-2015.html

```diff
@@ -0,0 +1,75 @@
+title: Crate.IO Snowsprint 2015 recap: Dockerzeit
```

Comment:
titles style headlines - could not find a standard for that. but something line that:
- Capitalize the first word of the title/heading and of any subtitle/subheading;
- Capitalize all “major” words (nouns, verbs, adjectives, adverbs, and pronouns) in the title/heading, including the second part of hyphenated major words (e.g., Self-Report not Self-report); and
- Capitalize all words of four letters or more.